server/src/main.rs

```diff
@@ -97,6 +100,8 @@ impl Cli {
                 webserver::main(self.port).unwrap();
             }
             Commands::Start => {
+                println!("Starting capture Server...");
```

Comment:
Start the capture task from within the webserver startup code (`webserver.rs run_server`).